server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -71,40 +96,39 @@ public County county() {
   public CountyStatus status() {
     return my_status;
   }
-  
+
   /**
-   * @return the audit board dashboard for this county.
+   * Sets the dashboard status.
+   * 
+   * @param the_status The new status.
    */
-  public AuditBoardDashboard auditBoardDashboard() {
-    return my_audit_board_dashboard;
+  public void setStatus(final CountyStatus the_status) {
+    my_status = the_status;
   }
   
   /**
-   * Sets new set of CVRs, replacing any current set of CVRs.
-   * 
-   * @param the_cvrs The CVRs.
+   * @return the CVR upload timestamp. A return value of null means
+   * that no CVRs have been uploaded for this county.
    */
-  public void setCVRs(final Collection<CastVoteRecord> the_cvrs) {
```

Comment:
It's superfluous - can be obtained from a straightforward database query given the timestamp/countyID pair. So if we need to send the actual CVRs as part of the state we send the client, we'll build that state using this object and database queries.